### PR TITLE
Trunk warnings

### PR DESCRIPTION
**Pull request by Algunenano**
Fix for https://trac.osgeo.org/postgis/ticket/4527

### CODE REVIEW

**Comment by strk** [comment]
How about fixing the sign-compare warning rather than hiding it ?

**Comment by Algunenano** [comment]
> How about fixing the sign-compare warning rather than hiding it ?

It's completely under Postgresql code so we can't fix it from Postgis. The coding standards between the 2 projects are different, and AFAIK PG doesn't care about these kind of errors.

File: postgis/lwgeom_functions_basic.c [hunk 1]
```diff
@@ -1546,12 +1546,12 @@ Datum LWGEOM_makepoly(PG_FUNCTION_ARGS)
 		holes = lwalloc(sizeof(LWLINE *) * nholes);
 		for (i = 0; i < nholes; i++)
 		{
-#if GCC_VERSION >= 40604
```
Comment (pramsey) [inline postgis/lwgeom_functions_basic.c]: If you're stripping out use of `GCC_VERSION` (which is fine, I just am too dim to do the major/minor/patch logic), can you also remove the definition from postgis_config.h.in?

Comment (Algunenano) [reply postgis/lwgeom_functions_basic.c]: I don't see any `GCC_VERSION` in my `postgis_config.h.in` or anywhere in the tree. Did you add it and forgot to push it?

**Comment by pramsey** [comment]
I was wondering why you were opposed enough w/ my commit to strip it back out, but that explains it. Yeah, I didn’t commit that with the #defines. Sorry. Thanks for the change.

**Pull request closed without merging**